From 0f8b2a1c1f4c42e952dd9fba033db0ebd21470f3 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 18 May 2023 09:02:21 +1100
Subject: [PATCH 7/7] be more lenient with abseil version

we carry the patch for the only pertinent difference
between .2 & .3 anyway
---
 src/google/protobuf/port_def.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/google/protobuf/port_def.inc b/src/google/protobuf/port_def.inc
index 1c6a24945..2ee003e15 100644
--- a/src/google/protobuf/port_def.inc
+++ b/src/google/protobuf/port_def.inc
@@ -216,7 +216,7 @@ static_assert(PROTOBUF_CPLUSPLUS_MIN(201402L), "Protobuf only supports C++14 and
 #define PROTOBUF_ABSL_MIN(x, y) 1
 #endif
 
-static_assert(PROTOBUF_ABSL_MIN(20230125, 3),
+static_assert(PROTOBUF_ABSL_MIN(20230125, 2),
               "Protobuf only supports Abseil version 20230125.3 and newer.");
 
 // Future versions of protobuf will include breaking changes to some APIs.
